examples/resources/saml-connector.yaml

```
@@ -28,4 +28,6 @@ spec:
     - access
   # Provides a path to the IdP metadata.
   entity_descriptor_url: https://example.okta.com/app/your-app-id/sso/saml/metadata
+  # Optional SAML Single Logout endpoint. If set, upon logging out of Teleport, users will also be signed out of the SAML provider session.
```

```suggestion
  # Optional SAML Single Logout endpoint. If set, logging out of Teleport
  # will also log the user out of the SAML provider session.
```

Simplifies language and makes sure the comment doesn't wrap like it does in your preview screenshot.